src/vision/status_quo/barbara_wants_to_use_ghostcell.md

```diff
@@ -0,0 +1,274 @@
+# Barbara wants to use GhostCell-like cell borrowing with futures
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the
+brainstorming period. It is derived from real-life experiences of
+actual Rust users and is meant to reflect some of the challenges that
+Async Rust programmers face today.
+
+If you would like to expand on this story, or adjust the answers to
+the FAQ, feel free to open a PR making edits (but keep in mind that,
+as they reflect peoples' experiences, status quo stories [cannot be
+wrong], only inaccurate). Alternatively, you may wish to [add your own
+status quo story][htvsq]!
+
+## The story
+
+Barbara quite likes using statically-checked cell borrowing.  This is
+implemented in various ways in
+[GhostCell](http://plv.mpi-sws.org/rustbelt/ghostcell/) and
+[`qcell`](https://docs.rs/qcell/0.4.1/qcell/).  She would like to use
+statically-checked cell borrowing within futures, but there is no way
```

Comment:
It would be good to show why there is no way to get the "owner borrow" through the `Future::poll` call here. This is explained a little bit further down, but I was confused for a little while until I got to that part.

Reply:
Okay, I will expand that part